src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java

```diff
@@ -432,7 +433,7 @@ public boolean searchString(String toSearch, String query) {
 			}
 			return false;
 		}
-		if (lastStringMatch != -1 && NotEnoughUpdates.INSTANCE.config.hidden.showSearchbarDebug) {
+		if (lastStringMatch != -1 && NotEnoughUpdates.INSTANCE.config.hidden.debugFlags.contains(NEUDebugFlag.SEARCH)) {
```

Comment:
You can just use `NEUDebugFlag.SEARCH.isSet()`

Reply:
how was I supposed to know ;-;